po/ca.po

```diff
@@ -7,82 +7,83 @@ msgstr ""
 
 #: src/MainWindow.vala:76
 msgid "Calculating checksum…"
-msgstr ""
+msgstr "Calculant checksum…"
```

Comment:
Before gerunds, the Softcatalà style guide [prescribes the addition of “S'està...”](https://www.softcatala.org/guia-estil-de-softcatala/tota-la-guia/#Gerundi). In addition, you didn’t translate “checksum”, which is “[suma de comprovació](https://www.softcatala.org/recursos/web_search.py?source=checksum&target=&project=tots)” in Catalan.

Reply:
I did not know the existence of this guide. As a Catalan speaker, I used to translate without any guidance, using only my knowledge of the language. From now on, I will use it to improve my translations. Thank you.